capellambse/model/modeltypes.py

```diff
@@ -174,3 +175,25 @@ class Kind(StringyEnum):
     FIRMWARE = _enum.auto()
     PERSON = _enum.auto()
     SOFTWARE_APPLICATION = _enum.auto()
+
+
+class VisibilityKind(StringyEnum):
+    """Visibility kind"""
+
+    UNSET = _enum.auto()
+    PUBLIC = _enum.auto()
+    PROTECTED = _enum.auto()
+    PRIVATE = _enum.auto()
+    PACKAGE = _enum.auto()
+
+
+class CollectionKind(StringyEnum):
+    UNSET = _enum.auto()
+    ARRAY = _enum.auto()
+    SEQUENCE = _enum.auto()
+
+
+class UnionKind(StringyEnum):
+    UNSET = _enum.auto()
+    UNION = _enum.auto()
+    VARIANT = _enum.auto()
```

Comment:
Neither Collection nor Union allows `UNSET` as their `KIND`; they default to `ARRAY` and `UNION` respectively. These enums and the related properties need to be changed to reflect that.